### PR TITLE
Update stable to 1.9

### PR DESCRIPTION
**Pull request by Tomanhez**

### CODE REVIEW

**Comment by lchrusciel** [comment]
Can we migrate to GitHub actions?

**Comment by Tomanhez** [comment]
> Can we migrate to GitHub actions?

Yes I will do that in next PR

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 language: php
 
-dist: trusty
+dist: xenial
 
 sudo: false
 
 php:
-    - 7.3
+    - 7.4
```
Comment (GSadee) [inline .travis.yml:8]: Why only 7.4?

File: composer.json [hunk 2]
```diff
@@ -20,34 +20,34 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.4",
```
Comment (GSadee) [inline composer.json:23]: Why?

Comment (lchrusciel) [reply composer.json:23]: Probably it is our demo server version

File: composer.json [hunk 3]
```diff
@@ -20,34 +20,34 @@
...
+        "phpunit/phpunit": "^8.5",
         "stripe/stripe-php": "^4.1",
         "sylius-labs/coding-standard": "^2.0",
-        "symfony/browser-kit": "^3.4|^4.1",
-        "symfony/debug-bundle": "^3.4|^4.1",
-        "symfony/dotenv": "^3.4|^4.1",
-        "symfony/intl": "^3.4|^4.1",
-        "symfony/web-profiler-bundle": "^3.4|^4.1",
-        "symfony/web-server-bundle": "^3.4|^4.1"
+        "symfony/browser-kit": "^4.4",
```
Comment (lchrusciel) [inline composer.json:45]: Is it a big problem to update to 5.2?

File: composer.json (hunk 2 above)
Comment (Tomanhez) [reply composer.json]: Screenshot 2021-03-15 at 10 36 09

File: composer.json (hunk 3 above)
Comment (Tomanhez) [reply composer.json]: `symfony/browser-kit`,  `symfony/dotenv`, `symfony/intl` is possible to update to 5.2, but `symfony/debug-bundle`, `symfony/web-profiler-bundle`, `symfony/web-server-bundle` have conflict with few test libs 

Screenshot 2021-03-15 at 11 04 37


after update symfony-depedency-injection to higher version, I have other problems with for example: `friends-of-behat/exclude-specifications-extension`

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, Tomasz! :1st_place_medal:

**Pull request merged**